src/main/java/edu/cornell/kfs/sys/service/impl/AwsSecretServiceImpl.java

```diff
@@ -0,0 +1,164 @@
+package edu.cornell.kfs.sys.service.impl;
+
+import java.text.ParseException;
+import java.util.Date;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import com.amazonaws.ClientConfiguration;
+import com.amazonaws.PredefinedClientConfigurations;
+import com.amazonaws.client.builder.AwsClientBuilder;
+import com.amazonaws.services.secretsmanager.AWSSecretsManager;
+import com.amazonaws.services.secretsmanager.AWSSecretsManagerClientBuilder;
+import com.amazonaws.services.secretsmanager.model.DecryptionFailureException;
+import com.amazonaws.services.secretsmanager.model.EncryptionFailureException;
+import com.amazonaws.services.secretsmanager.model.GetSecretValueRequest;
+import com.amazonaws.services.secretsmanager.model.GetSecretValueResult;
+import com.amazonaws.services.secretsmanager.model.InternalServiceErrorException;
+import com.amazonaws.services.secretsmanager.model.InvalidParameterException;
+import com.amazonaws.services.secretsmanager.model.InvalidRequestException;
+import com.amazonaws.services.secretsmanager.model.LimitExceededException;
+import com.amazonaws.services.secretsmanager.model.MalformedPolicyDocumentException;
+import com.amazonaws.services.secretsmanager.model.PreconditionNotMetException;
+import com.amazonaws.services.secretsmanager.model.ResourceExistsException;
+import com.amazonaws.services.secretsmanager.model.ResourceNotFoundException;
+import com.amazonaws.services.secretsmanager.model.UpdateSecretRequest;
+import com.amazonaws.services.secretsmanager.model.UpdateSecretResult;
+import com.google.gson.Gson;
+
+import edu.cornell.kfs.sys.jsonadapters.JsonDateSerializer;
+import edu.cornell.kfs.sys.service.AwsSecretService;
+
+public class AwsSecretServiceImpl  implements AwsSecretService{
+    private static final Logger LOG = LogManager.getLogger(AwsSecretServiceImpl.class);
+    
+    protected String awsRegion;
+    protected String kfsInstanceNameSpace;
+    protected String kfsSharedNamespace;
```

Comment:
The capitalization in the "namespace" substring is inconsistent between these two. It would be helpful to have them consistent with each other.

Reply:
ok